ci-operator/config/openshift/release/openshift-release-master__ci-4.12.yaml

```diff
@@ -186,7 +186,7 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-azure-sdn-techpreview-serial
-- as: e2e-azure-sdn-upgrade
+- as: e2e-azure-sdn-upgrade-dnm-test-name
```

Comment:
Why the job rename? This would have pretty broad impacts and result in disjoint job history. Suspect this was for debugging?

Reply:
yuck. I'm sorry. I did not even see this comment for some reason.